Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/stationai.yml

```diff
@@ -48,49 +22,68 @@
             icon: Structures/Wallmounts/posters.rsi/poster11_legit.png
             iconOn: Structures/Wallmounts/posters.rsi/poster11_legit.png
             keywords: ["AI", "console", "interface", "laws", "borg"]
-            priority: -3
+            priority: -2
             event: !type:ToggleIntrinsicUIEvent
         - key: enum.CommunicationsConsoleUiKey.Key
           toggleAction:
             name: action-name-show-communications-console
             description: action-description-show-communications-console
-            icon: Structures/Machines/parts.rsi/box_0.png
-            iconOn: Structures/Machines/parts.rsi/box_2.png
+            icon: SimpleStation14/Mobs/Silicon/ai.rsi/icon-comms.png
+            iconOn: SimpleStation14/Mobs/Silicon/ai.rsi/icon-comms.png
             keywords: ["AI", "console", "interface"]
-            priority: -10
+            priority: -1
             event: !type:ToggleIntrinsicUIEvent
     - type: CommunicationsConsole
       title: Station AI
       color: "#93e2ff"
       delay: 5
       global: true
-    - type: InnateTool
-      tools:
-        - id: AIPDA
-        - id: DoorRemoteAI
+    # Innate door remotes (bolting, emergency access, etc)
     - type: InnateItem
```

Comment:
I suppose it doesn't have a PDA anymore, are all announcements just 'Unknown,' or did you manage to fix that?

Reply:
I want to remove that entirely, I'll look into it

Reply:
![image](https://user-images.githubusercontent.com/77995199/235404936-2abf93e1-4baf-44bc-a042-b6186cfe0f0d.png)
simple

Reply:
Having their name was nice 😢 
Oh well, this is fine

Reply:
when multiple AIs is a more realistic circumstance (not considering bureaucratic errors, they shouldn't be affected but are) I might add a name option to the AccessComponent and have it use that